.github/workflows/compile-latex.yml

```diff
@@ -0,0 +1,29 @@
+on:
+    pull_request: 
+        branches: 
+            - main
+        paths:
+            - 'docs/**'
```

Comment:
nice job! you almost succeeded in your first PR.

May you could add a tex file in the docs directory?

After that, there will be compiled pdfs in the github actions section.